src/modules/auth/controllers/auth.controller.ts

```diff
@@ -0,0 +1,56 @@
+import {
+  Body,
+  Controller,
+  HttpCode,
+  HttpStatus,
+  Post,
+  Req,
+  UseGuards,
+} from '@nestjs/common';
+import { JwtAtGuard } from 'src/common/guards/jwt-at.guard';
+import { JwtRtGuard } from '../../../common/guards/jwt-rt.guard';
+import { CreateUserDto } from '../../users/dtos/createUser.dto';
+import { AuthDto } from '../dtos/auth.dto';
+import {
+  RequestWithRefreshToken,
+  RequestWithToken,
+} from '../types/tokens.type';
+
+import { AuthService } from '../interfaces/AuthService';
+import type { Tokens } from '../types/tokens.type';
+
+@Controller('auth')
+export class AuthController {
+
+  constructor(private readonly authService: AuthService) {}
+
+  @Post('/signin')
+  @HttpCode(HttpStatus.CREATED)
+  public async signin(@Body() dto: AuthDto): Promise<Tokens> {
+    return await this.authService.signin(dto);
+  }
```

Comment:
```suggestion
  public async signin(@Body() dto: AuthDto): Promise<Tokens> {
    return this.authService.signin(dto);
  }
```

await не потрібен якщо ти в асинхронному методі повертаєш проміс